packages/firestore/src/remote/serializer.ts

```diff
@@ -887,30 +887,44 @@ export function toQueryTarget(
   return result;
 }
 
+export interface RunAggregationQueryRequestWrapper {
+  request: ProtoRunAggregationQueryRequest;
+  aliasMap: Record<string, string>;
+}
+
```

Comment:
IMO It's better not to introduce this interface and have the serializer function return 2 things.

Reply:
Do you know if all of our SDK platforms support tuple return values? I like this, but it would be nice to stay consistent where possible.